src/components/Recipes/RecipeDetail.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import RecipeDetail from './RecipeDetail'
+
+export default {
+  component: RecipeDetail,
+  title: 'RecipeDetail',
+  // Our exports that end in "Data" are not stories.
```

Comment:
brauchst du Zeile 7 + 8? Also hast du Dateien mit Data, die du nicht als Story exportieren möchtest?

Reply:
Jap. Die `const recipeDetailData` ab Zeile 11 möchte ich nicht als Story exportieren, sondern nur ihre Daten in der `const Recipe` in Zeile 40 verwenden.